src/tools/sfdk/command.h

```diff
@@ -51,6 +51,18 @@ class Domain;
 class Emulator;
 class Module;
 
+class Hook
```

Comment:
Why not struct? This looks a lot like a struct to me...

Reply:
It looks like a struct but it is not meant to be thought of as such. It is related to the Command, Option, Domain, Module, etc. classes - some look more like structs, some less, but all are used in the same context, so mixing class/struct would not be good.

Reply:
fair enough